wpl_cli/src/asset/instructions/royalty/instructions/modify.rs

```diff
@@ -0,0 +1,58 @@
+use anyhow::Result;
+
+use solana_program::system_program::ID as SYSTEM_PROGRAM_ID;
+use solana_sdk::{
+    compute_budget::ComputeBudgetInstruction,
+    message::{v0::Message as TransactionMessage, VersionedMessage},
+    signer::Signer,
+    transaction::VersionedTransaction,
+};
+use spl_token_2022::ID as TOKEN_PROGRAM_ID;
+use wen_new_standard::instructions::{ModifyRoyalties, ModifyRoyaltiesInstructionArgs};
+
+use super::super::RoyaltyArgs;
+use crate::{asset::parse_update_royalties_args, Context};
+
+pub async fn run(context: Context, args: RoyaltyArgs) -> Result<()> {
+    let Context { client, keypair } = context;
+    let payer = keypair.pubkey();
+    let recent_blockhash = client.get_latest_blockhash().await?;
+
+    let mint_pubkey = args.mint;
+    let keypair_pubkey = keypair.pubkey();
+
+    let modify_royalties = ModifyRoyalties {
+        payer: keypair_pubkey,
+        authority: keypair_pubkey,
+        mint: mint_pubkey,
+        token_program: TOKEN_PROGRAM_ID,
+        system_program: SYSTEM_PROGRAM_ID,
+    };
+
+    let update_royalties_args = parse_update_royalties_args(args.config_path)?;
+
+    let compute_budget_set_units_ix = ComputeBudgetInstruction::set_compute_unit_limit(300_000);
```

Comment:
Why are we bumping cu limit here?

Reply:
This technically is a monkey patch, but when users are adding royalties, the negative part of adding additional_metadata is they have to be done iteratively (one key value pair at a time), hence this CU would be enough for a 2 creator with royalty basis points. However this will be dynamically resized based on the transaction size in the upcoming PR.